src/lightning_app/components/multi_node/lite.py

```diff
@@ -0,0 +1,95 @@
+import os
+from dataclasses import dataclass
+from typing import Any, Callable, Type
+
+from typing_extensions import Protocol, runtime_checkable
+
+from lightning_app.components.multi_node.base import MultiNode
+from lightning_app.components.multi_node.pytorch_spawn import _PyTorchSpawnRunExecutor
+from lightning_app.core.work import LightningWork
+from lightning_app.utilities.app_helpers import is_static_method
+from lightning_app.utilities.packaging.cloud_compute import CloudCompute
+from lightning_app.utilities.tracer import Tracer
+
+
+@runtime_checkable
+class _LiteProtocol(Protocol):
```

Comment:
```suggestion
class _LiteWorkProtocol(Protocol):
```

in the beginning I thought this was to add a check for lite itself, which didn't make too much sense.
Should this maybe inherit from LightningWork and the Protocol ?

Reply:
Done !